src/test/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobViewTest.java

```diff
@@ -95,14 +98,14 @@ public void should_know_the_url_of_the_last_build() {
 
     @Test
     public void progress_of_a_not_started_job_should_be_zero() {
-        view = JobView.of(a(job()));
+        view = JobView.of(a(job()), config);
```

Comment:
I agree that `config` should be injected, but I'd rather like the injection being hidden in the helper method `of(..)` because:
- we're not doing anything with the `config` spy in the test; ie. we're not asserting on it
- accentuating the fact that the `config` object is injected doesn't improve the readability of the test case